docs/content/design/indexing-service.md

```diff
@@ -68,6 +68,17 @@ The overlord console can be used to view pending tasks, running tasks, available
 http://<OVERLORD_IP>:<port>/console.html
 ```
 
+#### Blacklisted Workers
+If the workers fail tasks above a threshold, the overlord will blacklist these workers. No more than 20% of the nodes can be blacklisted. Blacklisted nodes will be periodically whitelisted.
```

Comment:
is max percentage of blacklist workers configurable ? 
If not, can we make it configurable ?